src/Task/TYPO3/CMS/AbstractCliTask.php

```diff
@@ -202,4 +202,15 @@ protected function ensureApplicationIsTypo3Cms(Application $application)
                 1420210955);
         }
     }
+
+    /**
+     * @param array $options
+     *
+     * @return string
+     */
+    protected function getCliDispatchScriptFileName(array $options = array())
+    {
+        $webDirectory = isset($options['webDirectory']) ? trim($options['webDirectory'], '\\/') : '';
+        return $webDirectory !== '' ? sprintf('%s/typo3/cli_dispatch.phpsh', $webDirectory) : 'typo3/cli_dispatch.phpsh';
```

Comment:
The entry ponit `typo3/cli_dispatch.phpsh`is deprecated since TYPO3 v8, we need to keep that in mind.

Reply:
Any quick suggestions, what we can do? I think no one is using coreapi anyway in version TYPO3 8

Reply:
It won't be removed until TYPO3 9 anyway, so we should be fine with keeping it in until then. If it turns out that no one is using the Core API anyway, we could consider dropping support for it. 

I don't feel like there is any need to add for new functionality/configuration options here.